src/Client.php

```diff
@@ -226,7 +226,7 @@ protected function decodeJson($raw)
         $decoded = json_decode($raw);
         $err = json_last_error();
         if ($err !== JSON_ERROR_NONE) {
-            throw new InvalidJsonException($err);
+            throw new InvalidJsonException($raw);
```

Comment:
@Gman98ish Im a bit late to the party on this, but is there a reason you arent returning the error message?

would be nice to know why it fail as well as getting the raw response body.

https://www.php.net/manual/en/function.json-last-error-msg.php

Reply:
am thinking something like

`throw new InvalidJsonException(json_last_error_msg() . ': ' . $raw);`